docs/api/sequelize.md

```diff
@@ -984,7 +984,7 @@ An OR query
 ## `json(conditions, [value])` -> `Sequelize.json`
```

Comment:
Remove all changes from this file, its auto generated, change actual comments in same file `sequelize.js`